catalogue/webapp/components/Download/Download.js

```diff
@@ -106,116 +75,84 @@ const Download = ({
     >
       {downloadOptions.length > 0 && (
         <>
-          {isEnhanced ? (
-            <>
-              <h2 className="inline">
-                <ShameButton
-                  isDark={useDarkControl}
-                  ariaControls={ariaControlsId}
-                  ariaExpanded={showDownloads}
-                  onClick={() => {
-                    setShowDownloads(!showDownloads);
-                    setAlignmentOfDownloadOptions();
-                  }}
-                >
-                  <span className={`btn__text`}>Downloads</span>
-                  <Icon name={useDarkControl ? 'download' : 'chevron'} />
-                </ShameButton>
-              </h2>
-            </>
-          ) : (
-            <Space
-              v={{
-                size: 'l',
-                properties: ['margin-top'],
-              }}
-            >
-              <h2
-                className={classNames({
-                  [font('hnm', 5)]: true,
-                  'work-details-heading': true,
-                })}
-              >
-                Download
-              </h2>
-            </Space>
-          )}
-          <DownloadOptions
-            id={ariaControlsId}
-            className={classNames({
-              [font('hnm', 5)]: true,
-            })}
-            alignment={alignment}
-            aria-hidden={!showDownloads}
-            hidden={!showDownloads}
-            enhancedStyles={isEnhanced}
+          <DropdownButton
+            label="Downloads"
+            isInline={useDarkControl}
```

Comment:
why do we use useDarkControl to determine if it's inline?

Reply:
That's a great question – I think the answer is laziness (the only place where it is currently inline is in the viewer, so this was sufficient). I'll make it more declarative.